src/style.css

```diff
@@ -0,0 +1,110 @@
+/* stylelint-disable color-hex-case */
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.card-block {
+  box-sizing: border-box;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  background: white;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+
+}
+
+.mac-image {
+  display: block;
+  width: 160px;
+  height: 134px;
+  margin: 0 auto 40px;
+}
+
+.advert-title {
+  font-size: 12px;
+  line-height: 18px;
+  font-weight: 500;
+  color: #060B35;
+  margin: 0 0 4px;
+  font-style: medium;
+}
+
+.product-code {
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+  margin-bottom: 16px;
+}
+
+.product-review {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+  margin-bottom: 24px;
+
+}
+
+.stars-container {
+  display: flex;
+}
+
+.star {
+  background-image: url(./images/star-active.svg);
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+  margin-right: 4px;
+  background-position: center;
+}
+
+.star:last-child {
+  background-image: url(./images/star.svg);
+  margin-right: 0;
+}
```

Comment:
Here you should be able to update the number of stars in the review. According to the checklist:

> Reuse stars block from [Stars task](https://github.com/mate-academy/layout_stars) and keep it operational. If you change stars--4 modifier to stars--2 or other - number of selected stars should change correctly.